configure/recipes/timescale.rb

```diff
@@ -0,0 +1,47 @@
+unless node.attribute?(:ec2)
+  # Copy config files
+  cookbook_file "/etc/postgresql/12/main/pg_hba.conf" do
+    source "pg_hba.conf"
+    owner "root"
+    group "root"
+    mode "0644"
+  end
+  cookbook_file "/etc/postgresql/12/main/postgresql.conf" do
+    source "postgresql.conf"
+    owner "root"
+    group "root"
+    mode "0644"
+  end
+
+  # Tune pgsql for timescale
+  execute "tune-and-restart-pgsql" do
+    command "sudo timescaledb-tune -yes -quiet; sudo service postgresql restart"
+    user "root"
+  end
+
+  # restart pgsql
+  service "pgsql" do
+    supports status: true, restart: true, reload: true
+    action %i{stop disable}
+  end
+
```

Comment:
Try to register this service in `configure/services.rb` which loads all services that can be added/modified.
In `environments/dev.json` add the service state to `configure.services.pgsql`

Reply:
Further checks show our current pattern as being:
1. Define the service in `base/recipes/timescale.rb` with `action %i{stop disable}%`
2. edit_resource the service in `configure/recipes/services.rb`